src/actions/run-command.cpp

```diff
@@ -48,12 +48,19 @@ void RunCommand::executePrelude() {
   }
 }
 
-void RunCommand::executeAction(const Gesture& /*gesture*/) {
-  RunCommand::runCommand(this->command);
+void RunCommand::executeAction(const Gesture& gesture) {
+  RunCommand::runCommand(this->command, gesture);
 }
 
-void RunCommand::executeReverse(const Gesture& /*gesture*/) {
-  RunCommand::runCommand(this->decreaseCommand);
+void RunCommand::executeReverse(const Gesture& gesture) {
+  RunCommand::runCommand(this->decreaseCommand, gesture);
+}
+
+bool RunCommand::runCommand(const std::string& command,
+                            const Gesture& gesture) {
+  setenv("DEVICE_TYPE",
```

Comment:
I don't think this could cause side effects, but, do you think it is worth calling `unsetenv` afer calling `system`?